The libswmhack library is known to cause issues on GNU/Linux, so it doesn't
make much sense to build it.
Index: scrotwm/linux/Makefile
===================================================================
--- scrotwm.orig/linux/Makefile	2009-03-23 22:30:17.000000000 +0100
+++ scrotwm/linux/Makefile	2009-03-23 22:30:37.000000000 +0100
@@ -5,18 +5,10 @@
 
 CC= gcc
 
-LVERS= $(shell . ../lib/shlib_version; echo $$major.$$minor)
-
-all: scrotwm libswmhack
+all: scrotwm
 
 scrotwm: scrotwm.o linux.o
 	$(CC) $(LDFLAGS) -o $@ $+ $(LDADD)
 
-%.so: %.c
-	$(CC) $(CFLAGS) -c -fpic -DPIC $+ -o $@
-
-libswmhack: swm_hack.so
-	$(CC) -shared -fpic -o libswmhack.so.$(LVERS) `lorder swm_hack.so|tsort`
-
 clean:
-	rm -f scrotwm *.o *.so libswmhack.so.*
+	rm -f scrotwm *.o
